flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -0,0 +1,582 @@
+                                 output_num=output_num,
+                                 operation_type='predict',
+                                 **kwargs)
+
+
+def evaluate(input_data: Union[Channel, List[Channel]],
+             model_info: Union[ModelMeta, Text, int],
+             executor,
+             output_num=0,
+             name: Text = None,
+             **kwargs) -> Union[NoneChannel, Channel, Tuple[Channel]]:
+    """
+    Evaluate Operator. Do evaluate job with the specific model version.
+
+    :param input_data: List of Channel. It contains the dataset data used in prediction.
+    :param model_info: Information about the model which is in prediction. Its type can be ModelMeta
+                       of py:class:`ai_flow.meta.model_meta.ModelMeta` or Text or int. The model_info
+                       means name in he metadata service when its type is Text and it means id when its type is
+                       int. The ai flow will get the model meta from metadata service by name or id.
+    :param executor: The user defined function in evaluate operator. User can write their own logic here.
+    :param output_num: The output number of the operator. The default value is 0.
+    :param name: Name of the predict operator.
+    :return: NoneChannel.
+    """
+    if isinstance(model_info, ModelMeta):
+        model_meta = model_info
+    elif isinstance(model_info, Text):
+        model_meta = get_ai_flow_client().get_model_by_name(model_info)
+    else:
+        model_meta = get_ai_flow_client().get_model_by_id(model_info)
+
+    return user_define_operation(input_data=input_data,
+                                 name=name,
+                                 model_info=model_meta,
+                                 executor=executor,
+                                 output_num=output_num,
+                                 operation_type='evaluate',
+                                 **kwargs)
+
+
+def dataset_validate(input_data: Channel,
+                     executor,
+                     name: Text = None,
+                     **kwargs
+                     ) -> NoneChannel:
+    """
+    Dataset Validator Operator. Identifies anomalies in training and serving data in this operator.
+
+    :param input_data: Channel. It contains the dataset data used in evaluation.
+    :param executor: The user defined function in dataset validate operator. User can write their own logic here.
+    :param name: Name of the dataset validate operator.
+    :return: NoneChannel.
+    """
+    return user_define_operation(input_data=input_data,
+                                 executor=executor,
+                                 name=name,
+                                 output_num=0,
+                                 operation_type='dataset_validate',
+                                 **kwargs
+                                 )
+
+
+def model_validate(input_data: Union[Channel, List[Channel]],
```

Comment:
Can we also provide a typical / expected behavior here to report the chosen model version?